incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -20,59 +19,75 @@ const eventuallyTimeout = 5
 var hncRecoverPath = os.Getenv("HNC_REPAIR")
 
 func FieldShouldContain(resource, ns, nm, field, want string){
-	FieldShouldContainMultiple(resource, ns, nm, field, []string{want})
+	fieldShouldContainMultipleWithTimeout(1, resource, ns, nm, field, []string{want}, eventuallyTimeout)
 }
 
 func FieldShouldContainMultiple(resource, ns, nm, field string, want []string){
-	FieldShouldContainMultipleWithTimeout(resource, ns, nm, field, want, eventuallyTimeout)
+	fieldShouldContainMultipleWithTimeout(1, resource, ns, nm, field, want, eventuallyTimeout)
 }
 
 func FieldShouldContainWithTimeout(resource, ns, nm, field, want string, timeout float64){
-	FieldShouldContainMultipleWithTimeout(resource, ns, nm, field, []string{want}, timeout)
+	fieldShouldContainMultipleWithTimeout(1, resource, ns, nm, field, []string{want}, timeout)
 }
 
 func FieldShouldContainMultipleWithTimeout(resource, ns, nm, field string, want []string, timeout float64){
+	fieldShouldContainMultipleWithTimeout(1, resource, ns, nm, field, want, timeout)
+}
+
+func fieldShouldContainMultipleWithTimeout(offset int, resource, ns, nm, field string, want []string, timeout float64){
```

Comment:
Why do we want the 'offset', what's it used for?

Reply:
In the commit message, look at the filename and line numbers. In "before," they point to testutils.go, which isn't useful to figure out which part of the *test* failed. In "after," they identify the line of the test that failed.

The way we do that is to pass an offset to Gomega. This offset specifies the number of levels in the stack that it should pop to print the right filename and line number. See https://onsi.github.io/gomega/#making-assertions-in-helper-functions for more.